.github/actions/ci-setup/action.yml

```diff
@@ -10,6 +10,7 @@ runs:
       with:
         node-version: 20.x
         cache: 'pnpm'
+        registry-url: 'https://registry.npmjs.org'
```

Comment:
bizarre that we might need to configure this given this really is the default

Reply:
Yeah I don't understand why that'd be the case 😭